content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: MedicationAdministration | DSTU 2 API
+---
+
+# MedicationAdministration
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Administration resource is a record of the event of a patient consuming or otherwise being administered a medication.  Medications Administrations are assumed to be Given unless documented otherwise.
+<br/><br/>
+The following fields are returned if valued:
+
+* [Medication Administration id](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.identifier){:target="_blank"}
+* [Status](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.status){:target="_blank"}
+* [Patient](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.patient){:target="_blank"}
+* [Practitioner](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.practitioner){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.encounter){:target="_blank"}
+* [Prescription](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.prescription){:target="_blank"}
+* [Effective Time](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.effectiveTime_x_){:target="_blank"}
+* [Medication](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_){:target="_blank"}
+* Details of medication administered:
+  * [Dosage](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage){:target="_blank"}
+  * [Route](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.route){:target="_blank"}
+  * [Quantity](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.quantity){:target="_blank"}
+  * [Site](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.site_x_){:target="_blank"}
+  * [Rate](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.rate_x_){:target="_blank"}
+
+
+<!-- ## Terminology Bindings
+
+<%= terminology_table(:medication_statement, :dstu2) %> -->
```

Comment:
Do we really want to reuse the med statement definitions here?

Reply:
I think I was confused about Terminology Bindings, Medication Administration doesn't have any that I am aware of: https://github.com/cerner/fhir.cerner.com/pull/169/commits/037712040917c1d556c6c8c76363db6eb41412c2#diff-39322533700e1bb799f2f6d703a9ae74L32